DESCRIPTION

```diff
@@ -44,7 +44,8 @@ Suggests:
     knitr,
     rmarkdown,
     testthat (>= 2.1.0),
-    withr
+    withr,
+    tidyselect
```

Comment:
Let's omit this change.

Reply:
gotta have this change _if_ we are using tidyselect in vignette. If we're not using it then yes this should be dropped

Reply:
we certainly don't _need_ to use `tidyselect` in the vignette but it gives a nice pattern that I think scientists will like. If we don't think there's much issue with adding this dependency then let's go for it. 

For the record though, the `tidyselect` function in the vignette isn't even evaluated. It's in an `eval=F` block. Not sure if that makes a difference but it means that we have a dependency that we don't actually _use_ anywhere.

Reply:
Upon [further research](https://stackoverflow.com/a/9900160/3743213), seems like putting it in `Suggests` is the right move, so let's just do this.

Reply:
Agree that `Suggests` is a good place. Let's include that change in #158.